packages/react-native-sdk/src/components/LobbyView.tsx

```diff
@@ -81,53 +86,57 @@ export const LobbyView = () => {
-        <View style={styles.videoView}>
-          {isVideoAvailable ? (
-            <VideoRenderer
-              mirror={isCameraOnFrontFacingMode}
-              mediaStream={localVideoStream}
-              objectFit="cover"
-              style={styles.stream}
-            />
-          ) : (
-            <Avatar participant={connectedUserAsParticipant} />
-          )}
-          <ParticipantStatus />
-        </View>
-        <View style={styles.buttonGroup}>
-          <CallControlsButton
-            onPress={toggleAudioState}
-            color={muteStatusColor(isAudioMuted)}
-            style={[
-              styles.button,
-              theme.button.md,
-              {
-                shadowColor: muteStatusColor(isAudioMuted),
-              },
-            ]}
-          >
-            {MicIcon}
-          </CallControlsButton>
-          <CallControlsButton
-            onPress={toggleVideoState}
-            color={muteStatusColor(isVideoMuted)}
-            style={[
-              styles.button,
-              theme.button.md,
-              {
-                shadowColor: muteStatusColor(isVideoMuted),
-              },
-            ]}
-          >
-            {VideoIcon}
-          </CallControlsButton>
-        </View>
+        {connectedUser && (
+          <View style={styles.videoView}>
+            {isVideoAvailable ? (
+              <VideoRenderer
+                mirror={isCameraOnFrontFacingMode}
+                mediaStream={localVideoStream}
+                objectFit="cover"
+                style={styles.stream}
+              />
+            ) : (
+              <Avatar participant={connectedUserAsParticipant} />
+            )}
+            <ParticipantStatus />
+          </View>
+        )}
+        {connectedUser && (
+          <View style={styles.buttonGroup}>
+            <CallControlsButton
+              onPress={toggleAudioState}
+              color={muteStatusColor(isAudioMuted)}
+              style={[
+                styles.button,
+                theme.button.md,
+                {
+                  shadowColor: muteStatusColor(isAudioMuted),
+                },
+              ]}
+            >
+              {MicIcon}
+            </CallControlsButton>
+            <CallControlsButton
+              onPress={toggleVideoState}
+              color={muteStatusColor(isVideoMuted)}
+              style={[
+                styles.button,
+                theme.button.md,
+                {
+                  shadowColor: muteStatusColor(isVideoMuted),
+                },
+              ]}
+            >
+              {VideoIcon}
+            </CallControlsButton>
+          </View>
+        )}
         <View style={styles.info}>
           <Text style={styles.infoText}>
-            You are about to join a test call at Stream. 3 more people are in
-            the call now.
+            You are about to join a call with id {call?.id} at Stream. {count}{' '}
```

Comment:
Is mentioning call id necessary.. for example in google meet only how many are already in the call is shown..

And if count is 0 may be say.. you will be the first to join

Reply:
This text is tentative and according to design. I am not sure, why it is so.

Reply:
ah okay design seems incomplete then to me..

Reply:
upto you to change or not, was a suggestion